telugu/Telanganatelugu.md

```diff
@@ -0,0 +1 @@
+బతుకమ్మ - తెలంగాణ నేల పూల కవాతు అలికి ముగ్గులు పెట్టిన లోగిళ్ళు సింగారించుకున్న చెల్లెల్ల చేతి నైపుణ్యం వెలిసిన పూల పిరమిడ్ల ఇంధ్ర ధనువులు'' (నేల మీది చందమామలు, కాసుల లింగారెడ్డి) అని బతుకమ్మను తెలంగాణ సాంస్కృతిక చిహ్నంగా చేసుకుని నెత్తిన ఎత్తుకోవడంలో మిగతా ప్రాంతాల నుంచి వేరుపడి తన ప్రత్యేక అస్తిత్వాన్ని చాటుకోవడమే. ఇదంతా కవి అంతర్గత, బాహ్య వలసాధిపత్యాలను వ్యతిరేకించే క్రమంలో చేసే కవితా గానమే.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.